mri/reconstruct/utils.py

```diff
@@ -222,3 +247,23 @@ def generate_operators(data, wavelet_name, samples, mu=1e-06, nb_scales=4,
             plot_output=None)
 
     return gradient_op, linear_op, prox_op, cost_op
+
+
+def imshow3D(volume, display=True):
```

Comment:
I think this doesn't really belong in reconstruct. I think we could put it in a more general utils at the root level.